TREKPOLOGY/index.html

```
@@ -0,0 +1,8 @@
+<body>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)